src/test/java/teammates/test/cases/webapi/ActionFactoryTest.java

```diff
@@ -0,0 +1,53 @@
+package teammates.test.cases.webapi;
+
+import org.apache.http.client.methods.HttpGet;
+import org.apache.http.client.methods.HttpPost;
+import org.testng.annotations.Test;
+
+import teammates.common.exception.ActionMappingException;
+import teammates.common.util.Const;
+import teammates.test.cases.BaseTestCase;
+import teammates.test.driver.MockHttpServletRequest;
+import teammates.ui.webapi.action.Action;
+import teammates.ui.webapi.action.ActionFactory;
+import teammates.ui.webapi.action.GetAuthInfoAction;
+
+/**
+ * SUT: {@link ActionFactory}.
+ */
+public class ActionFactoryTest extends BaseTestCase {
+
+    @Test
+    public void testGetAction() throws Exception {
+        ActionFactory actionFactory = new ActionFactory();
+
+        ______TS("Action exists and is retrieved");
+
+        MockHttpServletRequest existingActionServletRequest = new MockHttpServletRequest(
+                HttpGet.METHOD_NAME, Const.ResourceURIs.URI_PREFIX + Const.ResourceURIs.AUTH);
+        // Use this value for testing
```

Comment:
This comment does not seem to be necessary because it does not add much information. You might want to remove this (and for the other tests below as well).